source/common/quic/platform/flags_impl.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+#include <string>
+
+#include "absl/container/flat_hash_map.h"
+#include "absl/synchronization/mutex.h"
+
+namespace quiche {
+
+const std::string QuicFlagPrefix = "envoy.reloadable_features.FLAGS_quic";
```

Comment:
This prefix seems different from the one used by EnvoyReloadableFeature

Not all flags with a FLAGS_quic should have equivalent runtime features, only the ones that have the FLAGS_quic_reloadable_flag_ prefix.